mail_embed_image/models/ir_mail_server.py

```diff
@@ -8,14 +8,16 @@
 from odoo.addons.base.ir.ir_mail_server import encode_header_param
 from werkzeug.test import EnvironBuilder
 from werkzeug.wrappers import Request as WerkzeugRequest
-from lxml.html.soupparser import fromstring
+try:
+    from lxml.html.soupparser import fromstring
+except(ImportError, IOError) as err:
+    logging.info(err)
```

Comment:
this should be debug as of https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#importerror